shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/helper/date/DateTime.kt

```diff
@@ -0,0 +1,17 @@
+package co.nimblehq.ic.kmm.suv.helper.date
+
+import kotlinx.datetime.*
+
+interface DateTime {
+
+    fun today(): LocalDate
+}
+
+class DateTimeImpl: DateTime {
+
+    override fun today(): LocalDate {
+        return Clock.System.todayIn(TimeZone.currentSystemDefault())
+    }
+}
```

Comment:
Do we need to build this helper for the next Integration task? I can't find its usage

Reply:
Yes, both platforms need to get `today` information. I believe this can belong to the backend task, so I created the shared code here.

Reply:
@suho I understand. In Android chapter, we will move this datetime setup to the integration tasks. It's not too complex and easy to review when it's coming along with its usage. 😛 

That's fine now 👍

Reply:
@suho I keep this issue open as I have a concern here, will you format all DateTime data before pushing them to ViewModel in each platform?
- if yes, this makes sense to keep the DateFormatter in the shared module.
- if not, I don't think we need it 💭 

Let's see if we need or don't need it in the next Integration PRs. Let's keep the current implementation 💪

Reply:
@luongvo Yes, I will format all DateTime data before pushing them to the view model. That's why I created a folder `helper` in the shared module. Any feedback is welcome in this case.
From the beginning, I did create a use case to get the date data. However, I think a helper is good in this case. Let's feedback in the next implementation